### PR TITLE
Added Vaibhav in setup() and assertNotEquals() in getItemById()

### PR DESCRIPTION
**Pull request by vvyas23**
Added changes in setup and getItemById()

### CODE REVIEW

**Comment by vvyas23** [comment]
Resolving conflicts

File: InventoryManagement/src/test/java/com/example/InventoryManagement/services/InventoryServiceImplTest.java [hunk 1]
```diff
@@ -10,7 +10,7 @@ class InventoryServiceImplTest {
 
     @BeforeEach
     void setUp() {
-        System.out.println("Shweta Vaibhav");
+        System.out.println("Shweta Vaibhav Bhor");
```
Comment (ShwetaBhor) [inline InventoryManagement/src/test/java/com/example/InventoryManagement/services/InventoryServiceImplTest.java:13]: Add City.

**Review by ShwetaBhor — CHANGES_REQUESTED** [review]

**Comment by vvyas23** [comment]
Added city Pune as per review changes requested

File: InventoryManagement/src/test/java/com/example/InventoryManagement/services/InventoryServiceImplTest.java (hunk 1 above)
Comment (ShwetaBhor) [reply InventoryManagement/src/test/java/com/example/InventoryManagement/services/InventoryServiceImplTest.java:13]: Looks ok to me ! 👍

**Pull request merged**